src/ipoplib.py

```diff
@@ -17,8 +17,124 @@
 import time
 import urllib2
 import keyring
+import base64
+import traceback
 
 from threading import Timer
+from keyring.util.escape import escape as escape_for_ini
+from keyring.py27compat import configparser
+import keyring.backend
+from keyring.backends.file import Encrypted,BaseKeyring
+from keyring.util import properties
+
+class IPOPEncryptedKeyring(Encrypted, BaseKeyring):
```

Comment:
I don’t comfortable with this class. We simply copied EncryptedKeyring from the keyring package and hardcode the key. I know hardcoding a key was what we planned to do, but this seems not good idea to copy a whole class from package source code and change a single line. Shouldn't there be better implementation?